lesson7/Program.cs

```diff
@@ -74,6 +77,7 @@ public static void PrintArr(int[,] arr)
                 { arr[i, 0] = arr[i - 1, 0]; }
```

Comment:
```suggestion
                arr[i, 0] = arr[i - 1, 0];
```
Зачем лишний хлам в коде? Если писать скобки, то развёрнуто. Либо не захламлять.